### PR TITLE
2022-08-03 release

### PR DESCRIPTION
**Pull request by t-young31**

### CODE REVIEW

File: changelogs/template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# EMAP Release
+
+**Date: __TODO: FIXME__ Changes in this release**
+
+---
+
```
Comment (stefpiatek) [inline changelogs/template.md:6]: The template should probably have the table of changed attributes so we don't have to look at an old one and copy it

File: changelogs/current_change_log.md [hunk 2]
```diff
@@ -4,9 +4,27 @@
...
+PatientCondition | resolutionDateTime |resolutionDatetime | ✓
+ConsultationRequest | statusChangeTime |statusChangeDatetime | ✓
+HospitalVisit | presentationDatetime | presentationTime | ✓
+HospitalVisit | admissionDatetime | admissionTime | ✓
+HospitalVisit | dischargeDatetime | dischargeTime | ✓
+LabSample | receiptAtLabDatetime | receiptAtLab | ✓
+LabSample | sampleCollectionDatetime | sampleCollectionTime | ✓
+LocationVisit | admissionDatetime | admissionTime | ✓
+LocationVisit | dischargeDatetime | dischargeTime | ✓
+VisitObservationType | creationDatetime | creationTime | ✓
```
Comment (stefpiatek) [inline changelogs/current_change_log.md:21]: worth having these as their SQL representation so:
`patient_condition | added_datetime | added_date_tiime`

Though I don't imagine we've been consistent with this - we are releasing a database so should use what the end user will be seeing

**Review by stefpiatek — APPROVED** [review]
Thanks for getting that in, we should be better at being consistent and I'm going to make your bear the brunt of that, sorry!

File: changelogs/current_change_log.md (hunk 2 above)
Comment (stefpiatek) [reply changelogs/current_change_log.md:21]: agreed!

**Comment by stefpiatek** [comment]
@t-young31 can't request your review but I _think_ I've added in all changes for this release - would be good to have your eyes on it to see if there's anything that we've missed

File: changelogs/current_change_log.md [hunk 3]
```diff
@@ -5,8 +5,46 @@
...
+#### PlannedMovement
+
+Tracks the final history for each planned movement within the hospital.
+
+| Name | Type | Description |
+|---| --- |---|
+| planned_movement_id | bigint | Unique identifier in EMAP for this PlannedMovement record. |
+| hospital_visit_id | HospitalVisit | Identifier for the HospitalVisit associated with this record. |
+| location_id | Location | Planned Location  to move to, may be null. |
+| event_type | varchar(255) | /brief The type of planned movement event (ADMIT, TRANSFER, DISCHARGE). |
```
Comment (t-young31) [inline changelogs/current_change_log.md:19]: Do we want the `/brief` in these descriptions?

Comment (stefpiatek) [reply changelogs/current_change_log.md:19]: oh interesting, we don't, but I thought they were picked up by the documentation only - is there something different in the new config file?

Comment (t-young31) [reply changelogs/current_change_log.md]: noice

**Review by skeating — APPROVED** [review]

**Comment by t-young31** [comment]
> ...would be good to have your eyes on it to see if there's anything that we've missed

I went through the release InformDB diff and checked everything was added. LGTM, once the conflict has been resolved ⚔️

**Pull request merged**